pages/whats-new/2020-11-17-batch-imports-and-dashboard.mdx

```diff
@@ -2,7 +2,7 @@
 title: "Batch Imports and Updated Dashboard"
 date: "2020-11-17"
 tags: ["new"]
-description: "Batch Imports and updated dashboard"
+description: "We've revamped a bunch of how Grouparoo works under the hood, simplifying how our data imports work, and more."
```

Comment:
"Bunch" of what - perhaps: `We've revamped how Grouparoo works under the hood, simplifying how our data imports work, and more.`